From 806b6e01506dea4eae68ac3e468d9398d63cafcb Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 21 Jul 2011 10:50:40 +0100
Subject: [PATCH] build: Allow 'make quickcheck' test-tool args to be
 overridden.

A typical use for this is for packagers who want to increase
the default timeout:

  make quickcheck QUICKCHECK_TEST_TOOL_ARGS="-t 300"
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e1ddf0d..ca31727 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -219,7 +219,7 @@ CLEANFILES = \
 # is NOT a substitute for proper testing!
 
 quickcheck:
-	./run test-tool/libguestfs-test-tool
+	./run test-tool/libguestfs-test-tool $(QUICKCHECK_TEST_TOOL_ARGS)
 
 # Make binary distribution.
 
-- 
1.7.5.2

